src/constants/success.constant.ts

```diff
@@ -20,5 +20,9 @@ export const successMessage = (key: string) => {
 	message.loading({ content: "Loaded", key: key });
 };
 
+export const successCopyAddress = (address: string) => {
+	message.success("Address [" + address + "] copied!");
+};
+
```

Comment:
let's create a new function for success messages and in that we should have one for copy, we can even use the `openSuccessNotificationWithIcon` for this.